docs/reference/quickstart/aggs-tutorial.asciidoc

```diff
@@ -0,0 +1,530 @@
+          "sum": {
+            "field": "taxful_total_price"
+          }
+        },
+        "smoothed_revenue": { <2>
+          "moving_fn": { <3>
+            "buckets_path": "daily_revenue", <4>
+            "window": 3, <5>
+            "script": "MovingFunctions.unweightedAvg(values)" <6>
+          }
+        }
+      }
+    }
+  }
+}
+----
+// TEST[skip:Using Kibana sample data]
+<1> Calculate daily revenue first
+<2> Create a smoothed version of the daily revenue
+<3> Use `moving_fn` for moving window calculations
+<4> Reference the revenue from our date histogram
+<5> Use a 3-day window — use different window sizes to see trends at different time scales
+<6> Use the built-in unweighted average function
+
+.Example response (truncated)
+[%collapsible]
+====
+[source,console-result]
+----
+{ ...
+  "aggregations": {
+    "daily_sales": {
+      "buckets": [
+        {
+          "key_as_string": "2024-10-24T00:00:00.000Z", <1>
+          "key": 1729728000000,
+          "doc_count": 146, <2>
+          "daily_revenue": {
+            "value": 10578.53125 <3>
+          },
+          "smoothed_revenue": {
+            "value": null <4>
+          }
+        },
+        {
+          "key_as_string": "2024-10-25T00:00:00.000Z",
+          "key": 1729814400000,
+          "doc_count": 153,
+          "daily_revenue": {
+            "value": 10448
+          },
+          "smoothed_revenue": {
+            "value": 10578.53125 <5>
+          }
+        },
+        ...
+----
+// TEST[skip:Using Kibana sample data]
+<1> Date of the bucket in ISO format
+<2> Number of orders for this day
+<3> Raw daily revenue before smoothing
+<4> First day has no smoothed value as it needs previous days for the calculation
+<5> Moving average starts from second day, using a 3-day window
+====
+
+[TIP]
+====
+Notice how the smoothed values lag behind the actual values - this is because they need previous days' data to calculate. The first day will always be null when using moving averages.
+====
+
+[discrete]
+[[aggregations-tutorial-cumulative]]
+==== Track running totals
+
+[source,console]
+----
+GET kibana_sample_data_ecommerce/_search
+{
+ "size": 0,
+ "aggs": {
+   "daily_sales": {
+     "date_histogram": {
+       "field": "order_date",
+       "calendar_interval": "day"
+     },
+     "aggs": {
+       "revenue": {
+         "sum": {
+           "field": "taxful_total_price"
+         }
+       },
+       "cumulative_revenue": { <1>
+         "cumulative_sum": { <2>
+           "buckets_path": "revenue" <3>
+         }
+       }
+     }
+   }
+ }
+}
+----
+// TEST[skip:Using Kibana sample data]
+<1> Name for our running total
+<2> `cumulative_sum` adds up values across buckets
+<3> Reference the revenue we want to accumulate
+
+[discrete]
+[[aggregations-tutorial-next-steps]]
+=== Next steps
+
+Refer to the <<search-aggregations,aggregations reference>> for more details on all available aggregation types.
+
+//TODO add relevant links judiciously
```

Comment:
Good to see we agree about the need for lots of links to supporting content. Of course, I see this after leaving lots of comments about where links are needed 🫠

Reply:
Sorry I should have been clearer about that in the PR description, my bad 🙏